src/fobi/views/class_based.py

```diff
@@ -215,13 +313,21 @@ def get_template_names(self):
                 self.theme, self.get_theme_template_name())
         return [self.template_name]
 
-class FobiFormRedirectMixin(FormMixin):
-    obj = None
+class FobiFormRedirectMixin(FormMixin):    
+    object = None    
```

Comment:
@jstacoder:

My only remark is: try to avoid (where possible, where not driven by underlying framework/libraries) using of reserved names (in this case - `object`).

Reply:
I agree, which was why I initially went with obj, but Django and FormMixin want the views “object” to be called “object”, it got weird with obj because the underlying mixin expected object

Reply:
Fair enough. In that case it falls under the definition of `where not driven by underlying framework/libraries`. Thank you!